tests/common/plugins/conditional_mark/tests_mark_conditions.yaml

```diff
@@ -116,9 +116,9 @@ bgp/test_bgp_multipath_relax.py:
 
 bgp/test_bgp_queue.py:
   skip:
-    reason: "Unsupported topology"
+    reason: "Unsupported topology or mellanox T1 asic"
     conditions:
-      - "topo_type in ['m0', 'mx']"
+      - "topo_type in ['m0', 'mx'] or ((topo_type in ['t1', 't1-lag', 't1-64-lag', 't1-56-lag'])and (platform in ['x86_64-mlnx_msn2700-r0', 'x86_64-mlnx_msn2700a1-r0']))"
```

Comment:
Do we see the issue on all t1 topo?

Reply:
we see this issue on interfaces not part of lag. This is observed on T1 testbeds.